src/gossip/fuzz.zig

```diff
@@ -306,7 +303,7 @@ pub fn run() !void {
         allocator,
         fuzz_contact_info,
         fuzz_keypair,
-        entrypoints,
+        (&entrypoint)[0..1],
```

Comment:
never seen that syntax before - thats cool

Reply:
Yeah, it's semi-recent I believe. I believe it was added in 0.12.